docs/advanced/mbed_targets.md

```diff
@@ -141,13 +147,11 @@ Some mbed targets require specific actions for generating a binary image that ca
 }
 ```
 
-In this example, after the initial binary image for the target is generated, the build system will call the function `binary_hook` in the `TEENSY3_1Code` class. 
-
-<span class="notes">**Note:** The definition of the `TEENSY3_1Code` class **must** exist in the *targets.py* file. </span>
+After the generation of an initial binary image for the `TEENSY3_1`, the build system calls the function `binary_hook` in the `TEENSY3_1Code` class within `tools/targets/__init__.py`. 
 
-Because `toolchains` is also specified, `binary_hook` will only be called if the toolchain used for compiling the code is either `ARM_STD`, `ARM_MICRO` or `GCC_ARM`. Note that specifying `toolchains` is optional: if it's not specified, the hook will be called no matter what toolchain is used.
+The `TEENSY3_1` `post_binary_hook` only is called if the toolchain that compiles the code is `ARM_STD`, `ARM_MICRO` or `GCC_ARM`.
```

Comment:
Query: How can we change "is called" to active voice? What's calling it? You, the user?